content/docs/operating/configuration.md

```diff
@@ -220,13 +220,14 @@ A `tls_config` allows configuring TLS connections.
 ```
 
 
-### `<target_group>`
+### `<static_config>`
 
-A `target_group` allows specifying a list of targets and a common label set for them.
-They are the canonical way to specify static targets in a scrape configuration.
+A `static_confic` allows specifying a list of targets and a common label set
```

Comment:
We keep a note of the old name in the docs for a few months, to reduce the inevitable support hassles.